tokenizers/src/pre_tokenizers/byte_level.rs

```diff
@@ -510,4 +546,30 @@ mod tests {
             "Hello there dear friend! [PA D]"
         );
     }
+
+    #[test]
+    fn deserialization() {
+        // Before use_regex
+        let byte_level: ByteLevel = serde_json::from_str(
+            r#"{"type": "ByteLevel", "add_prefix_space": true, "trim_offsets": false}"#,
+        )
+        .unwrap();
+        // true
```

Comment:
redundant no ?

Reply:
To check we have properly loaded thee default value ? No, we should check the default is correct.

Reply:
This test is to check backward compatbility, we need to ensure the previously existing files are properly loaded basically.

Reply:
Resolving to keep discussion in one place.

Reply:
I meant the comment !

Reply:
Done !